modules/exploits/multi/php/car_rental_system.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Car Rental System 1.0 reverse shell',
+        'Description' => %q{
+          Car Rental System CMS 1.0, this module exploits a vulnerabilit, which allows to enter Admin page with SQL injection, and upload a PHP file and run a reverse shell.
+        },
+        'Author' =>
+          [
+            'Jyotsna Adhana and Saurav Shukla', # Discovery
+            'Fortunato Lodari - foxlox - fortunato dot lodari at dedagroup dot it' # metasploit module
+          ],
+        'References' =>
+          [
+            [ 'EDB', '48931' ],
+            [ 'URL', 'https://www.sourcecodester.com/php/14544/car-rental-management-system-using-phpmysqli-source-code.html' ],
+            [ 'URL', 'https://www.sourcecodester.com/download-code?nid=14544&title=Car+Rental+Management+System+using+PHP%2FMySQLi+with+Source+Code' ]
+          ],
+        'Privileged' => true,
+        'Stance' => Msf::Exploit::Stance::Aggressive,
+        'Platform' => ['php'],
+        'Arch' => [ARCH_PHP],
+        'Targets' =>
+          [
+            ['Automatic', {}]
+          ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'PrependMigrate' => true,
```

Comment:
This option isn't relevant because PHP payloads don't support migrating. That's a Windows Meterpreter exclusive feature.